config/webpack.dev.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = Merge.smart(commonConfig, {
     path.resolve(__dirname, '../src/index.jsx'),
 
     // Uncomment this entrypoint to return static demo data instead of calling the data-api
-    // path.resolve(__dirname, '../src/demo/index.js'),
+    path.resolve(__dirname, '../src/demo/index.js'),
```

Comment:
Note t reviewers: I use vagrant so had to change these files often - will remove this file before merging.

Reply:
In the future, can you `git stash` these types of changes instead of including them in the PR? Having these changes in the PR is also causing one of the tests to fail, specifically overriding the `enableCodeManagementScreen` configuration.